systemtest/playlist.xml

```diff
@@ -2378,4 +2378,370 @@
 		</groups>
 	</test>
 	<!-- Tests above pertain to Java 9 Modularity -->
+	
+	<!-- The tests below are added to test concurrent scavenge on z/OS and z/Linux using OpenJ9 SDK -->	
+	<test>
+		<testCaseName>ClassLoadingTest_ConcurrentScavenge</testCaseName>
+		<variations>
+			<variation>-Xgcpolicy:gencon -Xgc:concurrentScavenge</variation>
+		</variations>
+		<command>export JAVA_HOME=$(JDK_HOME)$(D)  && \
+	perl $(TEST_RESROOT)$(D)stf$(D)stf.core$(D)scripts$(D)stf.pl  \
+	-test-root=$(Q)$(TEST_RESROOT)$(D)stf;$(TEST_RESROOT)$(D)openjdk-systemtest$(Q) \
+	-systemtest-prereqs=$(Q)$(TEST_RESROOT)$(D)systemtest_prereqs$(Q)  \
+	-java-args=$(Q)$(JVM_OPTIONS)$(Q) \
+	-results-root=$(REPORTDIR)  \
+	-test=ClassloadingLoadTest; \
+	$(TEST_STATUS)</command> 
+		<subsets>
+			<subset>SE80</subset>
+			<subset>SE90</subset>
+			<subset>SE100</subset>
+		</subsets>
+		<levels>
+			<level>sanity</level>
+		</levels>
+		<groups>
+			<group>system</group>
+		</groups>
+		<impls>
+			<impl>openj9</impl>
+		</impls>
+		<platformRequirements>os.zos,os.linux,arch.390</platformRequirements>
```

Comment:
I do not think we can use `os.zos,os.linux` as it is `and` relationship. For running on z/OS and z/Linux only, we can just use `arch.390`

Reply:
Good suggestion! Updated the PR, testing on zLinux in a personal build now. What is confusing is how it successfully excluded the test from Linux, despite the fact that the flags have an and condition between them. I'll update you of the results of the testing.

Reply:
@llxia : the test gets scheduled and run as planned on zLinux (using arch.390): https://urv-jenkins.canlab.ibm.com:9095/view/SV_Tests/job/test.personal/242/console